docs/source/index.rst

```diff
@@ -5,67 +5,62 @@
-architecture at its core.
 
-Quil programs can be executed on a downloadable **Quantum Virtual Machine** (QVM). This is a classical simulation of a
-quantum processor that can simulate various qubit operations. The Forest SDK comes with a QVM that you can run on your
-local machine. More information about the QVM can be found in the :ref:`qvm`.
+**A few terms to orient you as you get started with Forest:**
+
+- pyQuil is an open source Python library developed at Rigetti Computing that allows you to write programs for quantum computers.
+  The source is hosted on `github <http://github.com/rigetticomputing/pyquil>`_.
+- Quil, the Quantum Instruction Language, is the lower-level code that pyQuil gets compiled into. A full description of
+  Quil can be found in our whitepaper, `A Practical Quantum Instruction Set Architecture <https://arxiv.org/abs/1608.03355>`_.
+- quilc is the Quil Compiler that compiles pyQuil into Quil. The SDK includes quilc, which will enable you to compile your
+  pyQuil programs into executable Quil code.
+- The QVM is a simulator of our quantum computers. When you download the SDK, you’ll install the QVM and you will execute
+  Quil programs against it.
+- Forest is our software development kit, optimized for near-term quantum computers that operate as coprocessors, working in
+  concert with traditional processors to run hybrid quantum-classical algorithms. For references on problems addressable
+  with near-term quantum computers, see `Quantum Computing in the NISQ era and beyond <https://arxiv.org/abs/1801.00862>`_.
 
 Our flagship product, `Quantum Cloud Services <http://rigetti.com/qcs>`_ offers users an on-premise, dedicated access
 point to our quantum computers, and to a powerful 34-qubit Quantum Virtual Machine. This access point sits in a Virtual
 Machine, which we call a Quantum Machine Image. A QMI is bundled with the same downloadable SDK mentioned above, and a
 Command Line Interface (CLI), which is used for scheduling compute time on our quantum computers. To sign up for our
-waitlist, please click the link above. If need access to our quantum computers for research, please email
-support@rigetti.com.
-
-If you are already familiar with quantum computing, then feel free to proceed to :ref:`start`. If you're just getting
-started, try out our Introduction to Quantum Programming juypter notebook, and take a look at our :ref:`intro`, where we
-use Quil, and introduce the basics of quantum computing and the Quantum Abstract Machine on which it runs. You can run
-this notebook by going into your pyQuil directory and finding the source folder in docs. Then run
-
-.. code::`jupyter notebook intro_to_qc.ipynb`.
+waitlist, please click the link above. If need access to our quantum computers for research, please email support@rigetti.com.
 
 
-Try it out!
-
-.. [1] https://arxiv.org/abs/1608.03355
-
 Contents
 --------
 
 .. toctree::
    :maxdepth: 3
 
-   SDK_Download_Instructions
```

Comment:
if we're removing these from the toc we should remove the corresponding rst files

Reply:
Thanks, deleted.